packages/react/package.json

```diff
@@ -54,8 +54,8 @@
     "@storybook/react-webpack5": "^7.0.0-alpha.0",
     "@storybook/storybook-deployer": "^2.8.16",
     "contexture": "^0.12.19",
-    "contexture-client": "^2.50.2",
-    "contexture-elasticsearch": "^1.19.16",
+    "contexture-client": "^2.51.0",
+    "contexture-elasticsearch": "^1.22.0",
```

Comment:
No need to bump these. Changesets will take care of it

Reply:
these were left over from the previous bump I reverted to grab your new updates for the outside click push. Will clean it up.